packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -287,11 +287,13 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
 
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
   useEffect(() => {
-    copyPastePlugin.onInsertInlineImage = (imageAttributes: Record<string, string>) => {
-      const { id, src } = imageAttributes;
-      setInlineImageLocalBlobs({ ...inlineImageLocalBlobs, [id]: src });
-      onInsertInlineImage && onInsertInlineImage(imageAttributes);
-    };
+    if (onInsertInlineImage) {
```

Comment:
We should either have an else block here
```
if (onInsertInlineImage) {copyPastePlugin.onInsertInlineImage = (imageAttributes: Record<string, string>) => {
        const { id, src } = imageAttributes;
        setInlineImageLocalBlobs({ ...inlineImageLocalBlobs, [id]: src });
        onInsertInlineImage(imageAttributes);
      };
    }
} else {
copyPastePlugin.onInsertInlineImage = undefined
}
```
or move the if inside copyPastePlugin.onInsertInlineImage 
```
copyPastePlugin.onInsertInlineImage = (imageAttributes: Record<string, string>) => {
if (onInsertInlineImage) {
        const { id, src } = imageAttributes;
        setInlineImageLocalBlobs({ ...inlineImageLocalBlobs, [id]: src });
        onInsertInlineImage(imageAttributes);
}
      };
```
as otherwise the onInsertInlineImage is not reset correctly